tiup/tiup-command-mirror-set.md

```diff
@@ -0,0 +1,47 @@
+---
+title: tiup mirror set
+---
+
+# tiup mirror set
+
+The `tiup mirror set` command is used to switch the current mirror and supports two forms of mirrors: local file system and remote network.
+
+The address of the official mirror is `https://tiup-mirrors.pingcap.com`.
+
+## Syntax
+
+```sh
+tiup mirror set <mirror-addr> [flags]
+```
+
+`<mirror-addr>` is the mirror address, which has two forms:
+
+- Network address: start with `http` or `https`. For example, `http://172.16.5.5:8080`, `https://tiup-mirrors.pingcap.com`, etc.
+- Local file path: the absolute path of the mirror directory. For example, `/path/to/local-tiup-mirror`.
+
+## Option
+
+### -r, --root
+
+Specifies the root certificate.
+
+The root certificate of each mirror is different, and the root certificate is the most critical part of mirror security. When using network mirroring, it might suffer from man-in-the-middle attacks. To avoid such attacks, it is recommended to manually download the root certificate of the root network mirror to the local:
```

Comment:
```suggestion
As the most critical part of mirror security, the root certificate of each mirror is different from one another. When you use the network mirror, it might suffer from man-in-the-middle attacks. To avoid such attacks, it is recommended to manually download the root certificate of the root network mirror to the local:
```